lib/modules/ens/embarkjs.js

```diff
@@ -298,7 +185,10 @@ __embarkENS.lookup = function(address) {
     let resolverContract = new web3.eth.Contract(self.resolverInterface, resolverAddress);
     return resolverContract.methods.name(node).call();
   }).then((name) => {
-    if (name === "" || name === undefined) throw Error("ENS name not found");
+    if (name === "" || name === undefined) {
+      console.warn("ENS name not found");
+      return;
+    }
     return name;
```

Comment:
This return does nothing. You need a callback for lookup and then you can do `callback(null, name);`